source/module_hamilt_general/module_surchem/test/cal_epsilon_test.cpp

```diff
@@ -0,0 +1,135 @@
+#ifdef __MPI
+#include "mpi.h"
+#include "../../../module_basis/module_pw/test/test_tool.h"
```

Comment:
Do we need to include this header file?

Reply:
Are you referring to “test_tool.h"? Indeed not necessary, and just delete it, thanks.

Reply:
Yes, I thinks it is unnecessary.